webrtc.html

```diff
@@ -7150,6 +7151,31 @@ <h2>Methods</h2>
                 </li>
               </ol>
             </dd>
+            <dt><code>setTargetJitterBufferDelay</code></dt>
+            <dd>
+              <p>Asynchronously provides a preference for the target jitter
+              buffer delay in seconds. The user agent SHOULD aim to match this
+              preference but due to congestion control and other network related
+              reasons the actual delay MAY be different.</p>
+              <p>When the <dfn data-idl id=
+              "widl-RTCRtpReceiver-setTargetJitterBufferDelay-Promise-void">
+              <code>setTargetJitterBufferDelay()</code></dfn> method is called:
+              </p>
+              <ol>
+                <li>
+                  <p>Let <var>delay</var> be the target jitter buffer delay in
+                  seconds</p>
```

Comment:
This algorithm doesn't do anything. As written there's also no need to return a promise, since it's synchronous.

I see nothing here that wouldn't work as an attribute, e.g. `sender.latency = 0.5;` That would also address reading back the set value.

Reply:
Compare to:
```js
const parameters = receiver.getParameters();
parameters.latency = 0.5; 
await receiver.setParameters(parameters);
```
🤔

Reply:
What should happen if user specify negative delay? Should be rejected by setParameters then?

Reply:
Presumably the network stack and whatever controls the jitter buffer would live on a different thread than the main one, so specifying this as promise-based seemed like a good idea to me, if only the steps were updated to say stuff like "in parallel" and "update the underlying congestion control's preferred jitter buffer delay" or something like that. But you could make is synchronous and have the preference live in JS thread, but that would make it an implementation-detail how the preference is communicated to the network stack.

With regards to using get/setParameters(): These reflect negotiated parameters, and for the receiver there is no setParameters() method. The jitter buffer on the other hand is a local preference that is not negotiated. So it sounds like a different API to me, but I'm not against expressing this as a "receiver parameter" if we want to go that route.

Reply:
The only reason to return a promise would be if there's a step that must be done off main-thread that can fail. I don't see that here.